src/cards/bloat.ts

```diff
@@ -64,7 +64,7 @@ const spell: Spell = {
     manaCost: 15,
     healthCost: 0,
     expenseScaling: 1,
-    probability: probabilityMap[CardRarity.COMMON],
+    probability: probabilityMap[CardRarity.UNCOMMON],
```

Comment:
Bloat is one of the funnest spells and isn't super powerful (it doesn't scale well to late game).  I'm comfortable with it being COMMON or SPECIAL but UNCOMMON seems too rare especially because it's hardly useful late game

Reply:
Bloat doesn't seem like a "core" spell that should go along with stuff like slash and push, and I see it way more often than I'd like. It's not entirely applicable to most builds, and feels like the mechanics are special/interesting enough to warrant uncommon.

Maybe my real concern with bloat is how weak/messy it is in late game. It's generally the curse I want to have the least out of all of them, so I avoid picking it and maybe thats why it seems so common to me.